source/renderer/app/actions/wallets-actions.js

```diff
@@ -78,7 +78,9 @@ export default class WalletsActions {
   }> = new Action();
   saveQRCodeImageSuccess: Action<{ walletAddress: string }> = new Action();
   getAccountPublicKey: Action<{ spendingPassword: string }> = new Action();
-  copyPublicKey: Action<{ publicKey: string }> = new Action();
+  getICOPublicKey: Action<{ spendingPassword: string }> = new Action();
+  copyWalletPublicKey: Action<{ publicKey: string }> = new Action();
```

Comment:
Please rename `copyWalletPublicKey ` into `copyAccountPublicKey `

Reply:
This is going to be done in the scope of [DDW-709](https://jira.iohk.io/browse/DDW-709)

Reply:
Perfect 👌